storybook/src/components/DescriptionList/DescriptionList.stories.tsx

```diff
@@ -54,6 +54,33 @@ export const MultipleDescriptions: Story = {
   },
 }
 
+export const MultipleTerms: Story = {
+  args: {
+    termsWidth: 'md',
+    children: [
+      <DescriptionList.Group key={1}>
+        <DescriptionList.Term key={1}>Achternaam</DescriptionList.Term>
+        <DescriptionList.Description key={2}>
+          De naam die een persoon van zijn of haar ouders krijgt
+        </DescriptionList.Description>
+      </DescriptionList.Group>,
+      <DescriptionList.Group key={2}>
+        <DescriptionList.Term key={1}>Naam</DescriptionList.Term>
+        <DescriptionList.Term key={2}>Voornaam</DescriptionList.Term>
+        <DescriptionList.Term key={3}>Roepnaam</DescriptionList.Term>
+        <DescriptionList.Term key={4}>Bijnaam</DescriptionList.Term>
+        <DescriptionList.Description key={5}>
+          De naam waarmee een persoon wordt aangesproken
+        </DescriptionList.Description>
```

Comment:
This description sits next to the fourth term. I think we want it next to the first one. Giving a description in a group a `grid-row-start: 1` will fix this.

Reply:
Which makes me wonder. We don’t need the divs if we could ‘move up’ a description a couple of rows. CSS can count preceding nodes with the next-sibling combinator: (e.g. through `dt + dt + dd { grid-row: ‘move up 1’ }`, `dt + dt + dt + dd { grid-row: ‘move up 2’ }`. Do you see an approch for this? We can’t use negative row numbers because the term may not be the first in the list.

Reply:
Or just use subgrid on the group? It has a [93% support in the netherlands](https://caniuse.com/mdn-css_properties_grid-template-rows_subgrid) at the moment, it's use would only apply to larger screens and you could fallback (the group) to a single column like the native DL does.